src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,60 @@
+package racingcar.domain;
+
+import java.util.Random;
+
+import racingcar.exception.InvalidNameException;
+import racingcar.exception.NameLengthException;
+
+public class Car {
+    private static final int MIN_MOVEMENT_THRESHOLD = 4;
+    private static final int GO = 1;
+    public static final int START = 0;
```

Comment:
```suggestion
    private static final int START = 0;
```
외부로 노출되지 않는 상수는 private이면 좋을 것 같아요 😄